cssom-1/Overview.bs

```diff
@@ -2059,6 +2060,10 @@ attribute of the associated <a>CSS style sheet</a>.
 
 The <dfn attribute for=CSSImportRule>styleSheet</dfn> attribute must return the associated <a>CSS style sheet</a>.
 
+The <dfn attribute for=CSSImportRule>layerName</dfn> attribute may return the layer name declared in the at-rule itself,
```

Comment:
Let's keep the `[=layer name=]` link. Also, not `may`, should be a `must`.

Reply:
Done, thanks!